catalogue/webapp/components/WorkDetails/OnlineResources.tsx

```diff
@@ -39,12 +39,16 @@ const OnlineResources: FunctionComponent<Props> = ({ work }: Props) => {
   ] = useState(false);
   const firstOfRemainingOnlineResourcesRef = useRef<HTMLAnchorElement>(null);
   const showHideResourcesRef = useRef<HTMLButtonElement>(null);
-
+  const firstRender = useRef(true);
   useEffect(() => {
     if (isShowingRemainingOnlineResources) {
       firstOfRemainingOnlineResourcesRef?.current?.focus();
     } else {
-      showHideResourcesRef?.current?.focus();
+      if (!firstRender.current) {
+        showHideResourcesRef?.current?.focus();
+      } else {
+        firstRender.current = false;
```

Comment:
This is a bit misleading as it's not the first render, as it's on the `isShowingRemainingOnlineResources` dependency.

Maybe we could write a hook similar to [`useSkipInitialEffect`](https://github.com/wellcomecollection/wellcomecollection.org/blob/main/common/hooks/useSkipInitialEffect.ts), but rather `useFirstRenderEffect` or `useInitialEffect`?